django/contrib/auth/__init__.py

```diff
@@ -91,6 +93,28 @@ def authenticate(request=None, **credentials):
     )
 
 
+async def aauthenticate(request=None, **credentials):
+    """
+    If the given credentials are valid, return a User object.
+    """
```

Comment:
I'm very new to this code but I was thinking whether duplicating the docstrings from the sync functions is desired or not. I did a quick check in the existing code and I see a (soft?) pattern of having the async version reference the sync ones:

```
django/core/cache/backends/base.py:207:    async def aget_many(self, keys, version=None):
django/core/cache/backends/base.py-208-        """See get_many()."""
--
django/core/cache/backends/base.py:235:    async def aget_or_set(self, key, default, timeout=DEFAULT_TIMEOUT, version=None):
django/core/cache/backends/base.py-236-        """See get_or_set()."""
--
django/core/cache/backends/base.py:273:    async def aincr(self, key, delta=1, version=None):
django/core/cache/backends/base.py-274-        """See incr()."""
--
django/core/cache/backends/base.py:341:    async def aclear(self):
django/core/cache/backends/base.py-342-        return await sync_to_async(self.clear, thread_sensitive=True)()
--
django/core/cache/backends/base.py:360:    async def aincr_version(self, key, delta=1, version=None):
django/core/cache/backends/base.py-361-        """See incr_version()."""
```

Reply:
Yeah looks like you're right, I've updated the docstrings.